samples/csharp_dotnetcore/14.nlp-with-dispatch/appsettings.json

```diff
@@ -1,4 +1,33 @@
 {
-    "botFilePath": "nlp-with-dispatch.bot",
-    "botFileSecret": ""
-}
\ No newline at end of file
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+
+  "BotFramework": {
+    "AppId": "",
+    "Password": ""
+  },
+
+  "BotServices": {
+
+    "Luis-Home-Automation-AppId": "",
+    "Luis-Home-Automation-Authoringkey": "",
+    "Luis-Home-Automation-Region": "",
+
+    "QnA-sample-qna-kbId": "",
+    "QnA-sample-qna-endpointKey": "",
+    "QnA-sample-qna-hostname": "",
+
+    "Luis-Weather-AppId": "",
+    "Luis-Weather-Authoringkey": "",
+    "Luis-Weather-Region": "",
+
+    "Luis-nlp-with-dispatchDispatch-AppId": "",
+    "Luis-nlp-with-dispatchDispatch-Authoringkey": "",
+    "Luis-nlp-with-dispatchDispatch-Region": ""
```

Comment:
Can we add more depth here? Having this section be flat means we need to be very careful about naming. Perhaps something like:
- BotsServices is an array of service objects with type and name properties, in addition to AppId and other service-specific properties.

This would make the logic in the **BotServices** class clearer, too.

Reply:
Typically, you are not going to have a ton of entries here. We tried having more depth, but ultimately decieded to keep it simple based on the "Getting Started" spec. Ultimately, users should use Key Vault to keep their secrets.